hack/lib/util.sh

```diff
@@ -741,14 +744,15 @@ function kube::util::ensure-cfssl {
 
     chmod +x cfssl || true
     chmod +x cfssljson || true
+  fi
```

Comment:
imho we should put an else here that simply fails and echos an explanation that on non-amd64 you must install these yourself currently. otherwise unless we already returned above, on non-amd64 we will print the message below suggesting that we attempted and failed to download it, which is slightly misleading.

Reply:
or rather, we could actually undo all of the changes to this block and simple add a check for non-amd64 hosts after the check for already installed binaries, which will be much simpler.